{
  "modified": "2023-08-25T16:05:33Z",
  "published": "2020-10-22T02:47:40Z",
  "schema_version": "1.5.0",
  "id": "CVE-2020-27619",
  "aliases": [
    "PSF-2020-6"
  ],
  "details": "In Python 3 through 3.9.0, the Lib/test/multibytecodec_support.py CJK codec tests call eval() on content retrieved via HTTP.",
  "affected": [
    {
      "package": {
        "ecosystem": "DHI",
        "name": "python",
        "purl": "pkg:dhi/python"
      },
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3.6.13"
            },
            {
              "fixed": "3.7.10"
            },
            {
              "fixed": "3.8.7"
            },
            {
              "fixed": "3.9.1"
            },
            {
              "fixed": "3.10.0-alpha2"
            }
          ],
          "repo": "https://github.com/python/cpython"
        }
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://access.redhat.com/security/cve/CVE-2020-27619"
    },
    {
      "type": "REPORT",
      "url": "https://bugs.python.org/issue41944"
    }
  ]
}